lineman/app/components/user_avatar/user_avatar.haml

```diff
@@ -1,7 +1,5 @@
-.user-avatar{aria-hidden: 'true', class: '{{size}}'}
-  .avatar.avatar-initials{ng-if: 'user.avatarKind == "initials"', ng-class: '{ coordinator: coordinator }'}
+.user-avatar{aria-hidden: 'true', ng-class: '{"user-avatar--large": large, "user-avatar--small": small, "user-avatar--coordinator": coordinator}'}
```

Comment:
thank you.. we went through a bunch of css contortions and there are still knots to iron out. This is a good suggestion